File: Data Structures & Algorithms/string-encode-and-decode/submission-2.java
class Solution {

    public String encode(List<String> strs) {
        String ans="";
        for(String s:strs)ans+=""+s.length()+"#"+s;
        return ans;
    }

    public List<String> decode(String str) {
        int i=0;List<String>ans=new ArrayList<>();
        for(int j=0;j<str.length();){
            int len=0;i=j;
            while(str.charAt(i)!='#'){len=str.charAt(i)-'0'+10*len;i++;}
            ans.add(str.substring(i+1,i+len+1));
            j=i+len+1;
        }return ans;

    }
}

File: Data Structures & Algorithms/plus-one/submission-2.py
class Solution:
    def plusOne(self, digits: List[int]) -> List[int]:
        st=0
        for i in digits:
            st*=10;
            st+=i;
        st+=1
        st=str(st)
        l=list(st)
        ans=[]
        for i in l:
            ans.append(int(i))
        return ans
        # for i in range(len(digits)):
        #     digits[i]+=1
        # return digits